src/main/java/com/dataloom/datastore/directory/controllers/PrincipalDirectoryController.java

```diff
@@ -120,6 +120,21 @@ public Void removeRoleFromUser( @PathVariable( USER_ID ) String userId, @PathVar
         path = USERS + SEARCH + SEARCH_QUERY_PATH,
         produces = MediaType.APPLICATION_JSON_VALUE )
     public Map<String, Auth0UserBasic> searchAllUsers( @PathVariable( SEARCH_QUERY ) String searchQuery ) {
-        return userDirectoryService.searchAllUsers( searchQuery );
+
+        String wildcardSearchQuery = searchQuery + "*";
+        return userDirectoryService.searchAllUsers( wildcardSearchQuery );
+    }
+
+    @Override
+    @GetMapping(
+            path = USERS + SEARCH_EMAIL + EMAIL_SEARCH_QUERY_PATH,
+            produces = MediaType.APPLICATION_JSON_VALUE )
+    public Map<String, Auth0UserBasic> searchAllUsersByEmail( @PathVariable( SEARCH_QUERY ) String emailSearchQuery ) {
+
+        // to search by an exact email, the search query must be in this format: email.raw="hristo@kryptnostic.com"
+        // https://auth0.com/docs/api/management/v2/user-search#search-by-email
+        String exactEmailSearchQuery = "email.raw:\"" + emailSearchQuery + "\"";
```

Comment:
Where does this belong?

This `exactEmailSearchQuery` * **is** * the query param. Passing this into `userDirectoryService.searchAllUsers()` becomes `GET	/api/v2/users?q=email.raw:"foo@bar.com"`, which is what we want for searching by an exact email:

https://auth0.com/docs/api/management/v2/user-search#search-by-email